src/main/kotlin/no/nav/familie/ba/sak/kjerne/behandling/domene/Behandling.kt

```diff
@@ -322,7 +323,10 @@ enum class Behandlingsresultat(val displayName: String) {
     HENLAGT_AUTOMATISK_FØDSELSHENDELSE(displayName = "Henlagt avslått i automatisk vilkårsvurdering"),
     HENLAGT_TEKNISK_VEDLIKEHOLD(displayName = "Henlagt teknisk vedlikehold"),
 
-    IKKE_VURDERT(displayName = "Ikke vurdert")
+    IKKE_VURDERT(displayName = "Ikke vurdert");
+
+    fun kanIkkeSendesTilOppdrag(): Boolean =
+        this in listOf(FORTSATT_INNVILGET, AVSLÅTT, FORTSATT_OPPHØRT, ENDRET_UTEN_UTBETALING)
```

Comment:
Jeg synes ofte det fort blir forvirrende med negative boolske navn. Om man skal ha tak i de som kan sendes til oppdrag blir det `!kanIkkeSendesTilOppdrag`
```suggestion
    fun kanSendesTilOppdrag(): Boolean =
        this !in listOf(FORTSATT_INNVILGET, AVSLÅTT, FORTSATT_OPPHØRT, ENDRET_UTEN_UTBETALING)
```

Reply:
Da må det endres der det den brukes også

Reply:
Jeg tror det er lettere med kanIkkeSendesTilOppdrag fordi kodene som bruker dette flytter til et annet steg enn iverksettelse om den er true